doc/html5lib.rst

```diff
@@ -1,13 +1,8 @@
 html5lib Package
 ================
 
-:mod:`html5lib` Package
------------------------
```

Comment:
Why take the header out here?

Reply:
There are two headers in a row with exactly the same text.